<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupEditor</class>
 <widget class="QDialog" name="GroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Group Editor</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>Configuration of module group ...</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Readout config</string>
        </attribute>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <layout class="QHBoxLayout">
             <item>
              <widget class="QLabel" name="textLabel1_2">
               <property name="text">
                <string>PP0-Type:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="PP0Box">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Opto Test</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Opto Final</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>eBoc</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>keep module settings</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="spacer7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="moduleTable">
             <column>
              <property name="text">
               <string>Module SN</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Assy SN</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Assy pos.</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Group ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Module ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Slot</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>InLink</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>OutLink0</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>OutLink1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>OutLink2</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>OutLink3</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>331</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GroupEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
